src/components/Pricing/PricingTable/Plans.js

```diff
@@ -99,7 +57,8 @@ export const Scale = ({
                     <Section title="Pricing starts at" className="mt-auto">
                         <div className="flex justify-between items-center">
                             <Price>
-                                $2,000<span className="text-base opacity-50">/mo</span>
```

Comment:
Label is "Pricing starts at", so $2k is technically accurate here, which is why it's static. Since the pricing calculator is now in the modal, that's the only place the slider value lives.

Reply:
`SCALE_MINIMUM_PRICING` is a const. Its fixed value is set to 1000 on src/components/Pricing/constants.tsx line 3.